wrappers/python/openmm/app/simulation.py

```diff
@@ -252,23 +263,17 @@ def _simulate(self, endStep=None, endTime=None):
                     self._generate_reports(unwrapped, False)
     
     def _generate_reports(self, reports, periodic):
-        getPositions = False
-        getVelocities = False
-        getForces = False
-        getEnergy = False
-        for reporter, next in reports:
-            if next[1]:
-                getPositions = True
-            if next[2]:
-                getVelocities = True
-            if next[3]:
-                getForces = True
-            if next[4]:
-                getEnergy = True
-        state = self.context.getState(positions=getPositions, velocities=getVelocities, forces=getForces,
-                                      energy=getEnergy, parameters=True, enforcePeriodicBox=periodic,
-                                      groups=self.context.getIntegrator().getIntegrationForceGroups())
-        for reporter, next in reports:
+        '''
+        reports is a tuple of (reporter, nextReport)
+        reporter is the corresponding reporter object
+        nextReport is a description on what the reporter needs in the context. This can either be a dict, or a tuple (old).
+        '''
```

Comment:
Can you make the docstring follow numpy format?  See any of the other methods in the file for examples.